tests/resources/serializers/test_marcxml_serializer.py

```diff
@@ -206,120 +202,125 @@ def test_marcxml_serializer_full_record(
 
     # We are setting explicitly the order of the communities as it's required to match the expected data
     record.data["parent"]["communities"]["ids"] = [community.id, community2.id]
+    # TODO: This is cheating, try to set this in the `updated_full_record` method above
```

Comment:
I tried to create the custom field in the fixture higher up in this field, but it seems to be ignored, not 100% sure how to make it work yet.

Reply:
That's probably because the `thesis:` custom fields are not configured in the fixtures... We probably should add them, since I see other serializers that depend on them, but I also see that they similarly "cheat" like you did here by not fetching a full record from the database always.